app/web_ui/src/routes/(app)/settings/manage_tools/[project_id]/add_tools/+page.svelte

```diff
@@ -1,183 +1,145 @@
+      headers: [],
+      button_text: "Connect",
+    },
+  ]
+
+  let sections = [
+    {
+      category: "Sample Tools",
+      items: [
+        // TODO: Add more custom tool servers, pre-filled in.
+        ...sampleMcpServers.map((tool) => ({
+          ...tool,
+          on_click: () => connectRemoteMcp(tool),
+        })),
+      ],
+    },
+    {
+      category: "Custom Tools",
+      items: [
+        {
+          name: "Remote MCP Servers",
+          description:
+            "Connect to remote MCP servers to add tools to your project.",
+          href: `/settings/manage_tools/${$ui_state?.current_project_id}/add_tools/remote_mcp`,
+          button_text: "Connect",
+          on_click: () => {
+            goto(
+              `/settings/manage_tools/${$ui_state?.current_project_id}/add_tools/remote_mcp`,
+            )
           },
         },
-      )
-
-      if (api_error) {
-        throw api_error
-      }
-
-      if (data?.id) {
-        // Navigate to the tools page for the created tool
-        goto(
-          `/settings/manage_tools/${$page.params.project_id}/tool_servers/${data.id}`,
-        )
-      }
-    } catch (e) {
-      error = createKilnError(e)
-    } finally {
-      submitting = false
-    }
-  }
+      ],
+    },
+  ]
 </script>
 
-<AppPage
-  title="Connect Remote MCP Server"
-  sub_subtitle="Connect to a remote Model Context Protocol (MCP) server to add external
-        tools to your project."
->
-  <div class="max-w-2xl">
-    <FormContainer
-      submit_label="Connect"
-      on:submit={connect_remote_mcp}
-      bind:error
-      bind:submitting
-    >
-      <FormElement
-        label="Name"
-        id="name"
-        description="A name to identify this MCP server."
-        bind:value={name}
-        max_length={120}
-      />
-
-      <FormElement
-        label="Description"
-        inputType="textarea"
-        id="mcp_description"
-        description="A description of this MCP server for your reference."
-        optional={true}
-        bind:value={description}
-      />
-
-      <FormElement
-        label="Server URL"
-        id="mcp_server_url"
-        description="The URL of the remote MCP server."
-        placeholder="https://example.com/mcp"
-        bind:value={server_url}
-      />
-
-      <!-- Headers section -->
-      <FormElement
-        inputType="header_only"
-        label="Headers"
-        id="headers_section"
-        description="If the documentation for the server you're adding requires custom headers, enter them here."
-        info_description="These are usually not needed. Some MCP servers require custom headers, such as the 'Authorization' headers. Refer to the documentation for the server you're adding to see if they require headers."
-        value=""
-      />
+<AppPage title="Add Tools">
+  <div class="max-w-4xl mt-12 space-y-12">
+    {#each sections as section}
+      <div class="space-y-6">
+        <!-- Category Header -->
```

Comment:
I think I said to copy/paste, but going forward let's be better about code sharing. Should create a "settings_header.svelte" and "settings_item.svelte" UI components in `/lib/ui`, and use them in both places. P2 if you want to do in this PR or not, this isn't awful or anything.

It has potential of making code cleaner. Instead of fancy maps and dynamic building of lists, you can just put a bunch of components in order. Easier to read/write/customize.

 - Header, title = "Sample tools"
 - SettingsItem, title="Github", description="...", onclick={openTool("https..."}
 - SettingsItem, title="PostHog", description="...", onclick={openTool("https..."}
 - etc

You don't want to do that today because it would be a ton of repeated code, but with a component that's gone.